.github/workflows/certification.yml

```diff
@@ -245,10 +254,65 @@ jobs:
           exit 1
         fi
 
+    - name: Prepare Cert Coverage Info
+      run: |
+       mkdir -p path/to/cov_files
+       SOURCE_PATH_LINEAR=$(echo ${{ env.SOURCE_PATH }} |sed 's#/#\.#g') # converts slashes to dots in this string, so that it doesn't consider them sub-folders
+       echo "${{ env.cover_per_only }}" >> path/to/cov_files/$SOURCE_PATH_LINEAR.txt
+
+    - name: Upload Cert Coverage Artifact
+      uses: actions/upload-artifact@v3
+      with:
+        name: cert_cov
+        path: path/to/cov_files
+        retention-days: 1
+
+    - name: Component Coverage Discord Notification 
```

Comment:
See if this action can be figured not to report the entire test run as failed.

If not, we should use the Discord SDK manually and do something like
```
python sendDiscordNotification.py || true
```

Reply:
Sure, let me try to take that into account.

Reply:
Current commit should take this into account, with continue-on-error.

Reply:
Please re-review now. Discord issue / outage won't make Action fail.